### PR TITLE
DTSlav - a pseudo ircbot for #darktable channel on oftc

### PR DESCRIPTION
**Pull request by johnny-bit**
Since we can't reliably get dtslave on new irc channel, we have to use github actions. I'll spread this bot to other darktable-org repos.

### CODE REVIEW

File: .github/workflows/irc.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          mv -f current_commit recent_commit
+      - name: IRC notification
+        if: ${{ steps.gitlog.outputs.log != '' }}
+        uses: Gottox/irc-message-action@v2
+        with:
+          server: "irc.oftc.net"
+          channel: '#darktable'
+          nickname: dtslav
+          message: |-
+            Helo, there be ${{ steps.gitlog.outputs.llc }} new commit(s) to ${{ github.repository }}:
```
Comment (elstoc) [inline .github/workflows/irc.yml:58]: I would prefer something like the old dtslave did: `Showing last X commits to <repo>`

**Comment by elstoc** [comment]
If you want to make it run a bit faster, set fetch-depth to something more reasonable. If we're running it every 5 minutes I'd guess we could set it to 300 without risk of missing anything and that would almost certainly speed things up (since it wouldn't be downloading the integration test images).

**Comment by johnny-bit** [comment]
Let's say "every 10 min" and "fetch depth 500" should be sane even for most ambitious PRs merged within short timeframe.

**Comment by aurelienpierre** [comment]
Ok, let's bite the bullet. Thanks !

**Pull request merged**